src/views/login/CallsignSetupView.tsx

```diff
@@ -17,7 +17,7 @@ const CallsignSchema = yup.object().shape({
   callsign: yup
     .string()
     .required("Koodinimi on pakollinen")
-    .matches(CALLSIGN_REGEX, "Koodinimi on virheellinen")
+    .matches(CALLSIGN_REGEX, "Sallitut merkit: a-z, A-Z, 0-9")
```

Comment:
Regex now checks for length as well.
It could be changed to only validated allowed characters and have separate Yup checks for length.
This would provide separate errors for each

Reply:
fix'd